src/lv_conf_internal.h

```diff
@@ -2027,15 +2030,15 @@
     #ifdef CONFIG_LV_USE_FS_WIN32
         #define LV_USE_FS_WIN32 CONFIG_LV_USE_FS_WIN32
     #else
-        #define LV_USE_FS_WIN32 0
+        #define LV_USE_FS_WIN32   _MSC_VER ? 1 : 0 
```

Comment:
This `_MSC_VER` is reporting so many warnings, can you try to fix it?
Like the existence of a code like this writing above?
```c
#ifndef _MSC_VER
    #define _MSC_VER 0
#endif
```

Reply:
We can add it to beginning of `lv_conf_internal.h` (generated by https://github.com/lvgl/lvgl/blob/master/scripts/lv_conf_internal_gen.py)

Reply:
I did add this there